middleware/authentication.js

```diff
@@ -0,0 +1,25 @@
+const User = require('../models/User')
+const jwt = require('jsonwebtoken')
+const {UnauthenticatedError} = require('../errors')
+
+
+const auth = (req, res, next) => {
+    // Check header
+    const authHeader = req.headers.authorization
+    if(!authHeader || !authHeader.startsWith('Bearer ')) {
+        throw new UnauthenticatedError('Authentication invalid')
+    }
+    const token = authHeader.split('')[1]
```

Comment:
There is an error here. You should specify a space to split. Ex:  authHeader.split(' ')[1]

Reply:
I have corrected it